src/Context.js

```diff
@@ -0,0 +1,9 @@
+import React, { createContext, useState } from 'react';
+
+export const Context = createContext();
+
+const ContextProvidor = (props) => {
```

Comment:
Typo: ContextProvidor
```suggestion
const ContextProvider = (props) => {
```